src/PolyhedralGeometry/PolyhedralFan/properties.jl

```diff
@@ -459,7 +435,7 @@ function starsubdivision(PF::PolyhedralFan, n::Int)
     # identify all maximal cones in the new fan
     d = Polymake.polytope.dim(cone)
     newmaxcones = [Vector{Int64}(Polymake.row(maxcones, i)) for i in 1:(Polymake.nrows(maxcones)) if i!= n]
-    for subset in Oscar.Hecke.subsets(Vector{Int64}(nthmaxcone), d-1)
+    for subset in Hecke.subsets(Vector{Int64}(nthmaxcone), d-1)
```

Comment:
or even this?
```suggestion
    for subset in subsets(Vector{Int64}(nthmaxcone), d-1)
```

Reply:
No.
```
julia> subsets([1,2],2)
ERROR: UndefVarError: subsets not defined
Stacktrace:
 [1] top-level scope
   @ REPL[16]:1

julia> Hecke.subsets([1,2],2)
1-element Vector{Vector{Int64}}:
 [1, 2]
```

Reply:
We could put it here: https://github.com/oscar-system/Oscar.jl/blob/master/src/imports.jl.